### PR TITLE
Integration test: use /tmp

### PR DESCRIPTION
**Pull request by dcposch**
@feross

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -9,7 +10,7 @@ const APP_VERSION = require('../package.json').version
 
 const IS_TEST = isTest()
 const PORTABLE_PATH = IS_TEST
-  ? path.join(__dirname, '../test/tempTestData')
+  ? path.join(process.platform === 'windows' ? os.tmpdir() : '/tmp', 'WebTorrentTest')
```
Comment (feross) [inline src/config.js]: `windows` should be `win32`

File: test/config.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const path = require('path')
+const os = require('os')
 
-const TEST_DIR = path.join(__dirname, 'tempTestData')
+const TEST_DIR = path.join(process.platform === 'windows' ? os.tmpdir() : '/tmp', 'WebTorrentTest')
```
Comment (feross) [inline test/config.js]: Same comment as above

File: src/config.js (hunk 1 above)
Comment (mathiasvr) [inline src/config.js]: Why not use `os.tmpdir()` on all platforms?

Comment (feross) [inline src/config.js]: The Mac `os.tmpdir()` is different with each run of the app (you can test this!), which causes the screenshots to be different, which won't work for our integration tests

**Pull request merged**